db/seeds.rb

```diff
@@ -7,7 +7,15 @@
 #   Character.create(name: 'Luke', movie: movies.first)
 
 day_time = Time.new(2016,7,11,8,0,0,"+02:00")
-locations = ["Mordor", "Neverland", "Sherwood Forest", "Valhalla", "Voyager", "Winterfell", "Yellow Submarine"]
+locations = ["Mordor", "Neverland", "Sherwood Forest", "Valhalla", "Voyager", "Winterfell", "Yellow Submarine"].map do |n|
+  params = { title: n, capacity: rand(5.20), color: "#%06x" % (rand * 0xffffff) }
+  if (c = ConferenceRoom.find_by_title(n))
+    c.update_attributes!(**params)
```

Comment:
this is old style (and alias), you should use update/update!
`c.update!(params)`